modules/webbrowser/src/processors/webbrowserprocessor.cpp

```diff
@@ -239,6 +245,29 @@ void WebBrowserProcessor::OnLoadingStateChange(CefRefPtr<CefBrowser> browser, bo
 void WebBrowserProcessor::process() {
     // Vertical flip of CEF output image
     cefToInviwoImageConverter_.convert(renderHandler_->getTexture2D(), outport_, &background_);
+
+    // If any dataframes are connected, send their input in json to individual javascript functions
+    if (dataFrames_.isConnected() && dataFrames_.hasData()) {
+        auto frame = browser_->GetMainFrame();
+        size_t dataframeNr = 0;
+        for (auto dataFrame : dataFrames_.getVectorData()) {
+            using json = nlohmann::json;
```

Comment:
Converting a DataFrame to json format should be moved to a utility function. Idealy in the Plotting module.